src/styles/blocks/card.scss

```diff
@@ -0,0 +1,111 @@
+.card {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: $card-width;
+  background: hsl(0, 0%, 100%);
+  border: 1px solid hsl(0, 0%, 95%);
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  transition: all 300ms;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &:hover &__product-name {
+    color: rgb(52, 86, 139);
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    display: block;
+  }
+
+  &__product-data {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    width: 100%;
+    margin-top: 40px;
+  }
+
+  &__product-name {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $main-text-color;
+    transition: all 300ms;
+  }
+
+  &__product-code {
+    font-size: 10px;
+    line-height: 14px;
+    color: $secondary-text-color;
+    margin-top: 4px;
+  }
+
+  &__stars-and-reviews,
+  &__price {
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+  }
+
+  &__reviews {
+    font-size: 10px;
+    line-height: 14px;
+    display: flex;
+    align-items: flex-end;
+    text-align: right;
+    color: $main-text-color;
+  }
+
+  &__price-text {
+    font-size: 12px;
+    line-height: 18px;
+    color: $secondary-text-color;
+  }
+
+  &__price-value {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    color: $main-text-color;
+  }
+
+  &__buy-button {
+    width: 100%;
+    height: 40px;
```

Comment:
Consider use padding instead of hardcodding height

Reply:
done